sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -614,7 +614,7 @@ static int get_sdw_dai_link_num(struct snd_soc_acpi_mach_params *mach_params)
 static void init_dai_link(struct snd_soc_dai_link *dai_links,
 			  int id, char *name, int playback, int capture,
 			  struct snd_soc_dai_link_component *cpus,
-			  char *cpu_name,
+			  int cpu_num,
```

Comment:
Please state why this is better than current method in commit message.

Reply:
Thanks ,  I will add it in commit message